spinta/manifests/tabular/helpers.py

```diff
@@ -488,15 +481,17 @@ def read(self, row: Dict[str, str]) -> None:
             else:
                 # TODO: Detect if ref is a unit or an enum.
                 self.data['enum'] = row['ref']
-
         if dataset or row['source']:
             self.data['external'] = {
                 'name': row['source'],
                 'prepare': self.data.pop('prepare'),
             }
 
         self.state.model.data['properties'][row['property']] = self.data
-
+        # Decide if unique constrains for column are needed
+        # Needs to have in column name _unique
+        if '_unique' in row['type']:
+            DataType.schema['unique']['default'] = True
```

Comment:
You should never mutate global variables. And proper places for loading `unique` should be here:

https://github.com/atviriduomenys/spinta/blob/master/spinta/types/model.py#L183-L190

But instead of returning `str, list[str]` you should return `str, list[str], dict[str, str]` and add content of third argument into `data` which then will be used to set `prop.unique = data['unique']`.